spec/cnb/exchange_rates_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'webmock/rspec'
+
+RSpec.describe CNB::Application do
+  let(:rates) { CNB::ExchangeRates.new }
+  let(:currency) { 'AUD' }
+  let(:monthly_currency) { 'ALL' }
+  let(:date) { Date.parse('25.02.2014') }
+
+  describe '#rate_for' do
+    before do
+      url = CNB::RATES_URL + date.strftime('%d.%m.%Y')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.